install.sh

```diff
@@ -9,8 +9,9 @@ umask 002
 
 source "$(dirname $0)/install/_lib.sh"  # does a `cd .../install/`, among other things
 
-source dc-detect-version.sh
 source parse-cli.sh
+source dc-detect-version.sh
```

Comment:
sidecar to move this below CLI parsing, conceptually I think CLI parsing should always be first